docs/core/extensions/sslstream-best-practices.md

```diff
@@ -0,0 +1,99 @@
+---
+title: TLS/SSL Best Practices
+description: Learn best practices to using SslStream in .NET
+author: rzikm
+ms.author: rzikm
+ms.date: 10/25/2022
+---
+
+# TLS/SSL Best Practices
+
+[TLS (Transport Layer Security)](https://en.wikipedia.org/wiki/Transport_Layer_Security) is a cryptographic protocol designed to secure communication between two computers over the internet. The TLS protocol is exposed in .NET via the [`SslStream`](https://learn.microsoft.com/en-us/dotnet/api/system.net.security.sslstream) class.
+
+This article presents best practices for setting up secure communication between client and server and assumes .NET Core version 3.1 or later. For best practices for .NET Framework, see [Transport Layer Security (TLS) best practices with the .NET Framework](https://learn.microsoft.com/en-us/dotnet/framework/network-programming/tls).
```

Comment:
We cannot have links in them that contain the locale (i.e.; `/en-us/`), nor can we use fully qualified URLs. Please install and use the Learn Authoring Pack tool for Visual Studio Code to leverage the tooling support for issues like this. It can create links that are site relative with ease, and can be used to insert `xref`. That's the other concern with this PR, code references should use the `xref` syntax rather than the MD link syntax.

https://review.learn.microsoft.com/en-us/help/platform/docs-authoring-pack-overview?branch=main